include/wlr/types/wlr_input_inhibitor.h

```diff
@@ -0,0 +1,25 @@
+#ifndef _WLR_TYPES_INPUT_INHIBITOR_H
+#define _WLR_TYPES_INPUT_INHIBITOR_H
```

Comment:
We decided to use a convention without a leading underscore.

Reply:
Wait when did we decide that

Reply:
https://github.com/swaywm/wlroots/pull/157